en/docs/learn/configuring-a-read-only-ldap-user-store.md

```diff
@@ -1,22 +1,47 @@
 # Configuring a Read-only LDAP User Store
 
-!!! tip " Before you begin!"
-    
+WSO2 identity server uses a embedded Read/Write LDAP as the primary user store.
+This document will guide to you to change that to a Read Only LDAP user store.
+
+!!! tip "Before you begin!"  
```

Comment:
Didn't we decide to move this tip to the bottom of the page

Reply:
done